qa/tasks/util/__init__.py

```diff
@@ -216,7 +223,7 @@ def introspect_roles(ctx, logger, quiet=True):
     elif len(nodes_monitor) < 3:
         ctx['dev_env'] = True
     else:
-        ctx['dev_env'] = False    
+        ctx['dev_env'] = False
```

Comment:
Could you gather all these changes not related to the issue you're fixing to a separate commit.

Reply:
Done